src/System.Management.Automation/engine/CommandProcessorBase.cs

```diff
@@ -63,7 +55,7 @@ internal CommandProcessorBase(CommandInfo commandInfo)
 
         #region properties
 
-        private InternalCommand _command;
+        private InternalCommand _command = default!;
```

Comment:
We're using the null-forgiving operator `!` quite a bit in this file. Is there a reason we prefer this over marking the properties as nullable?

Reply:
It is design question. If a design intention is to disable null the implementation should follow the design. Review just should check the both.